hadoop-hdds/server-scm/src/test/java/org/apache/hadoop/hdds/scm/container/TestSCMContainerManager.java

```diff
@@ -71,6 +71,8 @@
   private static XceiverClientManager xceiverClientManager;
   private static String containerOwner = "OZONE";
```

Comment:
"OZONE": May be this could also use the existing constant or the newly added constant as requested in previous other comments

Reply:
This magic constant is used in multiple tests, I would suggest to resolve this in a separate JIRA after this one get merged, I have created https://issues.apache.org/jira/browse/HDDS-2378 to track this, and I am open to post a patch for that as well.

Reply:
Sounds good.